python/fledge/services/core/api/plugins/discovery.py

```diff
@@ -83,6 +84,8 @@ async def get_plugins_available(request: web.Request) -> web.Response:
         msg = "Fetch available plugins package request failed"
```

Comment:
is this a special case of logging warning? or it is being logged by internal interface

Reply:
This is the case when we have RunTimeError; especially when APT returns non zero exit code